src/main/scala/com/picnicml/doddlemodel/metrics/RegressionMetrics.scala

```diff
@@ -15,4 +17,27 @@ object RegressionMetrics {
       sqrt((d.t * d) / y.length.toDouble)
     }
   }
+
+  /** Mean absolute error */
+  object Mae extends Metric {
+
+    lazy val higherValueIsBetter: Boolean = false
+
+    override def apply(y: Target, yPred: Target): Double = {
+      val d = y - yPred
+      sum(abs(d)) / y.length.toDouble
```

Comment:
No need to store `y - yPred` in a variable here since it's only used once. I stored it in `rmse` to avoid recomputation.